.env.example

```
@@ -0,0 +1,2 @@
+OpenAI_API_KEY='xxx'
```

dotenv-linter (3.3.0)

[warning] [LowercaseKey] The OpenAI_API_KEY key should be in uppercase

(LowercaseKey)